code/__DEFINES/dcs/signals/signals_mob/signals_mob_simple.dm

```diff
@@ -13,3 +13,30 @@
 ///FROM mob/living/simple_animal/hostile/ooze/eat_atom(): (atom/target, edible_flags)
 #define COMSIG_OOZE_EAT_ATOM "ooze_eat_atom"
 	#define COMPONENT_ATOM_EATEN  (1<<0)
+
+//Mob ability signals /datum/action/cooldown/mob_cooldown
```

Comment:
Can you move a lot of these to their own file? Already not too happy about things like oozes being in here, now that I see it.

Something like signals_mob_abilities.dm or something.

Reply:
Also all of these need to document the proc they're called from, and what arguments are passed.